server/src/main/java/org/elasticsearch/health/HealthStatus.java

```diff
@@ -18,8 +18,9 @@
 
 public enum HealthStatus implements Writeable {
     GREEN((byte) 0),
-    YELLOW((byte) 1),
-    RED((byte) 2);
+    UNKNOWN((byte) 1),
```

Comment:
Should the `merge` method use UNKNOWN as the default instead of GREEN? It seems more accurate.

Reply:
This is a good point. IMO the `orElse` that sort of implies "default" is a bit artificial there (as we shouldn't be merging a list of empty statuses - it'd be good to have an assert that the list is not empty IMO)

Reply:
Decided to have this throw instead of return a default since I think GREEN and UNKNOWN are reasonably competitive for a default value. If there aren't any problems, then things are GREEN right? Or I guess we don't know of any problems, so UNKNOWN? Nah, throw an error. This shouldn't be happening.